Add docs/admin/sysctls.md

sttts opened the pull request:
Here is the proposal: https://github.com/kubernetes/kubernetes/pull/26057
Here is the feature: https://github.com/kubernetes/features/issues/34

<!-- Reviewable:start -->
This change is [<img src="https://reviewable.kubernetes.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1126)
<!-- Reviewable:end -->

ingvagabund commented on docs/admin/sysctls.md:
```diff
@@ -0,0 +1,123 @@
+
+A number of sysctls are _namespaced_ in today's Linux kernels. This means that
+they can be set independently for each pod on a node. Being namespaced is a
+requirement for sysctls to be accessibly in a pod context within Kubernetes.
```
s/accessibly/accessible

ingvagabund commented on docs/admin/sysctls.md:
```diff
@@ -0,0 +1,123 @@
+- `net.*`.
+
+Sysctls which are not namespaced are called _node-level_ and must be set
+manually by the cluster admin, either by means of the onderlying Linux
```
s/onderlying/underlying

derekwaynecarr commented on docs/admin/sysctls.md:
```diff
@@ -0,0 +1,123 @@
+
+Sysctls which are not namespaced are called _node-level_ and must be set
+manually by the cluster admin, either by means of the underlying Linux
+distribution of the nodes (e.g. via `/etc/sysctls.conf`) or using a DeamonSet
```
s/DeamonSet/DaemonSet

derekwaynecarr commented on docs/admin/sysctls.md:
```diff
@@ -0,0 +1,123 @@
+
+By far, most of the _namespaced_ sysctls are not necessarily considered _safe_.
+
+For Kubernetes v1.4, the following set of _safe_ sysctls is supported:
```
s/is/are

sttts replied:
Sure about this one? The set is singular.

derekwaynecarr commented:
few typos, but lgtm otherwise.

derekwaynecarr commented:
one last comment, squash if you agree, then LGTM

sttts commented:
Addressed last comment, squashed and rebased.

sttts commented:
@kubernetes/docs what's the merge policy here? Anybody can just merge with a lgtm?

devin-donnelly commented:
I can merge, but if this feature isn't present until the 1.4 release, it should not be merged until then.

I'll do a docs review in the meantime.

sttts commented:
- removed net.ipv4.tcp_max_syn_backlog to be inline with https://github.com/kubernetes/kubernetes/pull/32072
- rebased

@devin-donnelly will you cherry-pick onto the 1.4 branch? Or should I create another PR for that?

sttts commented:
Replaced by #1182
